src/composables/useCvssScoresModel.ts

```diff
@@ -32,13 +32,15 @@ export function useCvssScoresModel(flaw: Ref<ZodFlawType>) {
 
   const flawNvdCvss3 = computed(() => getCvssData('NIST', 'V3'));
 
+  const formatScore = (score: any) => score?.toFixed(1);
```

Comment:
Is score always gona be a number? In that case I think we could specify the type or in other cases cast to number to avoid issues on the `toFixed` method.
```suggestion
  const formatScore = (score: number) => score?.toFixed(1);
```